From 6c52aea3100dfa4cf4128fbdb2ade0c4ee1ca269 Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Fri, 16 Jan 2015 15:00:54 +0800
Subject: [PATCH 0805/1594] MLK-10058-1 ARM: imx6sx: add one pcie phy
 regulator in gpc dts

commit 5dc66d6c77f982a841204d7edd615810ce747def from
git://git.freescale.com/imx/linux-2.6-imx.git

Register one pcie phy regulator in gpc driver, so thus,
the gpc related operations when pcie phy is powered
on/off can be moved into gpc driver

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 617d3f2..6dceb15 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -824,6 +824,7 @@
 				fsl,mf-mix-wakeup-irq = <0x7c00000 0x3d00 0x0 0x400200>;
 				interrupt-parent = <&intc>;
 				clocks = <&clks IMX6SX_CLK_GPU>;
+				pcie-phy-supply = <&reg_pcie_phy>;
 				#power-domain-cells = <1>;
 			};
 
-- 
1.7.5.4

